pages/storefront/new.tsx

```diff
@@ -259,9 +261,23 @@ export default function New() {
         label="Page Description"
         rules={[{ required: true, message: 'Please enter a page description.' }]}
       >
-        <Input.TextArea />
+      <Input.TextArea />
       </Form.Item>
-    </>
+              <Form.Item
+                name={['meta', 'mint']}
+                label="Mint? Leave Default For SOL"
```

Comment:
Let's incorporate the SPL-token registry here. Give people a logo and official mint hash so they can confirm.

Reply:
agree, @dandlezzz can you link your code relevant again

Reply:
I'm also unsure re: the wum.bo stuff - remember I og wrote this on noah's request, so I'd like to additionally write in their stuff, but it's pretty easy once I solve for dependency hell